.circleci/config.yml

```diff
@@ -247,7 +247,8 @@ jobs:
             mkdir /logs
             . /opt/conda/etc/profile.d/conda.sh
             # Install
-            mamba env create -n esmvaltool -f environment_python.yml
+            conda update -y conda
```

Comment:
```suggestion
```
Would this also work?

Reply:
you mean not update conda? Maybe, but that never hurts, we always have an up to date conda methinks

Reply:
The container should be fairly up to date anyway, it's been released just 8 days ago: https://hub.docker.com/r/condaforge/miniforge3/tags

Reply:
let's remove it then

Reply:
works v well, bud https://app.circleci.com/pipelines/github/ESMValGroup/ESMValTool/14533/workflows/fdc7cb8d-7703-4143-94c3-58fdbd4cbe06/jobs/83228